src/main/java/com/revature/models/Batch.java

```diff
@@ -32,14 +33,15 @@
 	private static final long serialVersionUID = 1L;
 	
 	@Id
+	@Positive(message="Batch number must be positive.")
```

Comment:
What about the creation of new Batches? Is it not common to have zero as a placeholder?